frontend/src/components/Wheel/index.js

```diff
@@ -1,89 +1,72 @@
-/*
-import React from "react";
+import { useEffect, useState } from "react";
 import SpinButton from "./button";
 import "./wheel.css";
-import WheelComponent from "react-wheel-of-prizes";
 
-const CreateSVG = () => {
-  return (
-    <PieCircle
-      slices={[
-        { percentage: "0.10", color: "blue" },
-        { percentage: "0.20", color: "green" },
-        { percentage: "0.25", color: "red" },
-        { percentage: "0.30", color: "yellow" },
-      ]}
-    />
-  );
-};
-
-export const PieCircle = (props) => {
-  let totalPercentage = 0;
+const Wheel = () => {
+  const [wheelInfo, setWheelInfo] = useState([]);
```

Comment:
The default here is not a list anymore I believe, as you have changed that in the backend.

Reply:
Done